File: 0001 Code Chef/0021 Masterchief final/implementation/0009 dart/master_chef.dart
import 'dart:io';


void main(){
  String? stringInput = stdin.readLineSync();
  if(stringInput != null){
    int? test = int.tryParse(stringInput);
    if(test != null){
      int count = test;
      while(count > 0){
        String? stringInput = stdin.readLineSync();
        if(stringInput != null){
          int? rank = int.tryParse(stringInput);
          if(rank != null){
            if(rank <= 10 && rank >= 1){
              print("YES");
            }else{
              print("NO");
            }
          }
        }
        count = count - 1;
      }
    }
  }
}

File: 0001 Code Chef/0020 Masterchief final/implementation/0001 c/master_chef.c
#include <stdio.h>

int main(){
    int test = 0;
    scanf("%d",&test);
    while(test > 0){
        int rank;
        scanf("%d",&rank);
        if(rank <= 10){
            printf("YES\n");
        }else{
            printf("NO\n");
        }
        test = test - 1;
    }
    return 0;
}